modules/installation-osp-verifying-installation.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-user.adoc
+
+[id="installation-osp-verifying-installation_{context}"]
```

Comment:
I'd consider if the extra steps in modules/installation-complete-user-infra.adoc might be relevant so you don't need to make a new module.